Dfe.PrepareTransfers.Web/Services/DocumentGenerators/BenefitsGenerator.cs

```diff
@@ -0,0 +1,48 @@
+using Dfe.Academisation.ExtensionMethods;
+using Dfe.PrepareTransfers.Data.Models.Projects;
+using Dfe.PrepareTransfers.DocumentGeneration;
+using Dfe.PrepareTransfers.DocumentGeneration.Elements;
+using Dfe.PrepareTransfers.DocumentGeneration.Interfaces;
+using Dfe.PrepareTransfers.Helpers;
+using Dfe.PrepareTransfers.Web.Models;
+using Dfe.PrepareTransfers.Web.Models.ProjectTemplate;
+using Dfe.PrepareTransfers.Web.Pages.Projects.BenefitsAndRisks;
+using Microsoft.IdentityModel.Tokens;
+using System.Collections.Generic;
+
+namespace Dfe.PrepareTransfers.Web.Services.DocumentGenerators
+{
+    public static class BenefitsGenerator
+    {
+        public static void AddBenefits(DocumentBuilder builder, ProjectTemplateModel projectTemplateModel)
+        {
+            builder.ReplacePlaceholderWithContent("BenefitsInformation", build =>
+            {
+                build.AddTextHeading("Benefits", HeadingLevel.One);
+
+                if (projectTemplateModel.TransferBenefits.IsNullOrEmpty())
+                {
+                    build.AddTable(new List<TextElement[]>
+                    {
+                        new[] { new TextElement { Value = "N/A" } }
+                    });
+                }
+
+                else
+                {
+                    foreach (var item in projectTemplateModel.ListOfTransferBenefits)
+                    {
+
+                       var displayedOption = item == TransferBenefits.IntendedBenefit.Other ? $"Other: {projectTemplateModel.OtherIntendedBenefit}" : EnumHelpers<TransferBenefits.IntendedBenefit>.GetDisplayValue(item);
+
+                        build.AddTable(new List<TextElement[]>
```

Comment:
Same as I said for risks below, move foreach down, saves space in doc

Reply:
Ah I've just seen Dan G's screenshots and it renders as rows regardless, so I'm happy with that.